ui/main/src/assets/i18n/fr.json

```diff
@@ -22,19 +22,15 @@
             "selectProcessText": "Sélectionner un Processus",
             "selectStateText": "Sélectionner un État",
             "selectTagText": "Sélectionner une Étiquette",
-            "publishDateFrom": "PUBLIÉE À PARTIR DE",
-            "publishDateTo": "PUBLIÉE JUSQU'À",
-            "activeFrom": "ACTIVE À PARTIR DE",
-            "activeTo": "ACTIVE JUSQU'À",
             "invalidPublishStartDate": "La date 'Publiée à partir de' est invalide.",
             "invalidPublishEndDate": "La date 'Publiée jusqu'à' est invalide.",
             "invalidActiveStartDate": "La date 'Active à partir de' est invalide.",
             "invalidActiveEndDate": "La date 'Active jusqu'à' est invalide.",
             "publishEndDateBeforeStartDate": "La date 'Publiée jusqu'à' doit être postérieure à la date 'Publiée à partir de'.",
             "activeEndDateBeforeStartDate": "La date 'Active jusqu'à' doit être postérieure à la date 'Active à partir de'.",
```

Comment:
Following previous remarks , you do not need anymore these messages as  the date range picker should prevent having theses errors